src/Application/User/Command/Mail/SendConfirmationMailCommandHandler.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Application\User\Command\Mail;
+
+use App\Application\CommandBusInterface;
+use App\Application\MailerInterface;
+use App\Application\User\Command\CreateTokenCommand;
+use App\Domain\Mail;
+use App\Domain\User\Enum\TokenTypeEnum;
+use App\Domain\User\Exception\UserNotFoundException;
+
+final readonly class SendConfirmationMailCommandHandler
+{
+    public function __construct(
+        private CommandBusInterface $commandBus,
+        private MailerInterface $mailer,
+    ) {
+    }
+
+    public function __invoke(SendConfirmationMailCommand $command): void
+    {
+        $email = trim(strtolower($command->email));
```

Comment:
(Détail) Dans une PR précédente tu avais utilisé

```suggestion
        $email = $this->stringUtils->normalizeEmail($command->email);
```